modules/seqexec/model/src/main/scala/seqexec/model/enumerations.scala

```diff
@@ -93,6 +93,57 @@ object enumerations {
       "DARK" -> "Dark"
     )
 
+    val Niri: Map[String, String] = Map(
```

Comment:
The reason to have these hardcoded is that we can't pull ocs2 files into the client. We may want to get these from the gem db eventually but at the moment Niri isn't supported there